streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -92,7 +96,7 @@ public StandbyTask(TaskId id,
     public void commit() {
         log.debug("standby-task [{}] Committing its state", id());
         stateMgr.flush(processorContext);
-
+        stateMgr.checkpoint(Collections.<TopicPartition, Long>emptyMap());
```

Comment:
Yes, as there are no ackedOffsets for standby tasks. The `stateMgr` will use the restored offsets when checkpointing.